cumulusci/tasks/salesforce/update_profile.py

```diff
@@ -180,8 +167,13 @@ def _expand_profile_members(self, package_xml):
             raise CumulusCIException(
                 "The package.xml does not contain a Profiles member."
             )
```

Comment:
Can we add the Profiles section if it's missing instead of raising an exception?